src/main/java/com/twilio/http/HttpUtility.java

```diff
@@ -0,0 +1,32 @@
+package com.twilio.http;
+
+import com.twilio.Twilio;
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@UtilityClass
+class HttpUtility {
+    String getUserAgentString(final List<String>... userAgentExtensions) {
```

Comment:
Can we add a dedicated test case for this util ?

Reply:
I have covered in NetworkHttpClientTest (method: testGetUserAgentExtensible). As this utility is being called from httpclient implementations.
Should I write a separate file for that ?